src/components/modules/newsletter/newsletter.values.ts

```diff
@@ -0,0 +1,5 @@
+export const NEWSLETTER_URLS = {
+  normal: 'https://http-toolkit.mailcoach.app/subscribe/a63464bc-1d3f-4318-9229-91061d658373',
```

Comment:
Could we name it default instead?
```suggestion
  default: 'https://http-toolkit.mailcoach.app/subscribe/a63464bc-1d3f-4318-9229-91061d658373',
```